web/src/components/visually-hidden/visually-hidden.css

```diff
@@ -0,0 +1,17 @@
+/*
+  stolen from GOV.UK
+  https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/helpers/_visually-hidden.scss
```

Comment:
This is licensed under MIT which is allowed in our MPL

Reply:
Maybe "Based on" is a little less on the nose :thinking: